config/samples/kcp/managed_workspace/apibinding_permissionclaims.yaml

```diff
@@ -0,0 +1,22 @@
+# The intent of this file is to show what the APIBinding looks like if you run
+# a setup using permission claims instead of all the APIs provided by one
+# APIExport.
```

Comment:
You can move it up.

Reply:
Sorry, move what up?

Reply:
`APIExport` to the line above. Not that long and feels weird.